src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
+    private static final int def_capacity = 10;
+    private static final Object[] empty_arraydata = {};
+
+
```

Comment:
Remove redundant empty lines. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/collections/array-list.html#dont-begin-class-or-method-implementation-with-empty-line)
```suggestion
    private static final int def_capacity = 10;
    private static final Object[] empty_arraydata = {};
```